### ISSUE TITLE
[BUG] Unable to use ChatGPT when the extension is enabled as of Today.

### ISSUE THREAD

**Issue by Shnxxx** [saeedezzati/superpower-chatgpt#2]
Couldn't use the ChatGPT when the extension is enabled.

It says,
**"Our systems have detected unusual activity from your system. Please try again later."** 

Browser: Edge
Version : Version 118.0.2088.61 (Official build) (64-bit)
Extension Version: 5.7.5

I already tried turning off my other extensions and I narrowed it down to Superpower ChartGPT.

**Comment by Delphirier** [comment]
Yes! None of the other models besides 3.5 and 4 give me the unusual activity message. Very disappointing since my stuff is quite context heavy.
